tests/integration/test_watchman_make.py

```diff
@@ -0,0 +1,114 @@
+# vim: ts=4:sw=4:sts=4:et:
+# Copyright 2012-present Facebook, Inc.
+# Licensed under the Apache License, Version 2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+# no unicode literals
+
+import WatchmanTestCase
+import os
+import os.path
+import sys
+import re
+import subprocess
+import signal
+import time
+import uuid
+
+class TestWatchmanMake(WatchmanTestCase.WatchmanTestCase):
+
+    def setUp(self):
+        if sys.platform == 'win32':
+            self.skipTest('Continuous integration testing only in Linux / MacOS')
+        if sys.version_info >= (3, 0):
+            self.skipTest('watchman-make is only compatible with Python 2')
+        if self.transport != 'local' or self.encoding != 'bser':
+            self.skipTest('Testing only with local transport and bser encoding')
+        self.marker = 'target-' + str(uuid.uuid4())
+
+    def _run_test(self, args, behavior):
+        root = self.mkdtemp()
+        self.touchRelative(root, '.watchmanconfig')
+        fake_env = dict(os.environ)
```

Comment:
you can set `fake_env['WATCHMAN_SOCK'] = self.getClient().sockpath` here and avoid adding the `-U` option to `watchman-make`